Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM mcr.microsoft.com/dotnet/runtime:7.0 as build
+ARG VS_VERSION=1.19.0-rc.6
+ARG VS_VERSION_TYPE=unstable
+
+WORKDIR /tmp
+ADD https://cdn.vintagestory.at/gamefiles/${VS_VERSION_TYPE}/vs_server_linux-x64_${VS_VERSION}.tar.gz \
+    ./vs_server_linux-x64_${VS_VERSION}.tar.gz
+ADD https://github.com/anegostudios/VintagestoryServerArm64/raw/main/vs_server_linux-arm64_${VS_VERSION}.tar.gz \
+    ./vs_server_linux-arm64_${VS_VERSION}.tar.gz
+
+WORKDIR /opt/vintagestory
+RUN tar xzf /tmp/vs_server_linux-x64_${VS_VERSION}.tar.gz
+RUN rm -rf VintagestoryServer \
+    VintagestoryServer.deps.json \
+    VintagestoryServer.dll \
+    VintagestoryServer.pdb \
+    VintagestoryServer.runtimeconfig.json \
+    Lib
+RUN tar xzf /tmp/vs_server_linux-arm64_${VS_VERSION}.tar.gz
```

Comment:
this could be changed to `ADD ./server .` to just use `./server/*` files directly and get rid of the extra download above. However, then I get an error on startup:
```
vsserver_1  | Unhandled exception. System.BadImageFormatException: Bad IL format. The format of the file '/opt/vintagestory/VintagestoryServer.dll' is invalid.
vsserver_1  | Aborted (core dumped)
```

Reply:
what does it return when you do file "/opt/vintagestory/VintagestoryServer.dll"

you need to make sure that when using the repo files you replace the VintagestoryServer.dll that is in the official x64 release with the arm one

Reply:
@Th3Dilli , never mind, I hadn't installed `git-lfs` :man_facepalming: 

So that works, now it downloads the specified x64 tarball and copies over from `./server` in this repo.